CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+v1.0.2
+------
+* **Release date**: 2016-11-17.
+* **Requires:** [tileserver v1.0.0](https://github.com/mapzen/tileserver/releases/tag/v1.0.0) and [tilequeue v1.0.1](https://github.com/mapzen/tilequeue/releases/tag/v1.0.1) and [mapbox-vector-tile v1.0.0](https://pypi.python.org/pypi/mapbox-vector-tile/v1.0.0).
```

Comment:
> Why do we link to pypi here but in requirements we link to the Git repo?

No reason. Which do you prefer?

> Similarly, we link to require specific versions of tileserver and tilequeue.

> Should we have dev and prod install targets instead of defaulting to dev?

I would rather not even have these requirements files checked in these repos in the first place. AFAIK only the getting started instructions use them. Maybe the tests?